### PR TITLE
kibana 5.1.1

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by nijikon** [comment]
@ilovezfs Can't we actually use node from brew here?

**Comment by ilovezfs** [comment]
@nijikon looking back to https://github.com/Homebrew/legacy-homebrew/pull/46593 we'd probably want to consult @jasontedor on that.

File: Formula/kibana.rb [hunk 1]
```diff
@@ -15,8 +15,9 @@ class Kibana < Formula
   end
 
   resource "node" do
-    url "https://nodejs.org/dist/v7.2.0/node-v7.2.0.tar.xz" # N.B. includes vendored dependencies
-    sha256 "486d4db7ef659521ad2fafefca877638da07bef61e2aee090207ff52149294fb"
+    url "https://nodejs.org/dist/v7.2.1/node-v7.2.1.tar.xz" # N.B. includes vendored dependencies
```
Comment (jasontedor) [inline Formula/kibana.rb]: This should be v6.9.0, that's what this version of Kibana is tested against.

File: Formula/kibana.rb [hunk 2]
```diff
@@ -15,8 +15,9 @@ class Kibana < Formula
...
+    mirror "https://npm.taobao.org/mirrors/node/latest-v7.x/node-v7.2.1.tar.xz"
```
Comment (jasontedor) [inline Formula/kibana.rb]: Why do we need the mirror?

**Review by jasontedor — CHANGES_REQUESTED** [review]
Thanks @ilovezfs, I left a comment about the Node version.

**Comment by jasontedor** [comment]
> Can't we actually use node from brew here?

Elastic tests and supports against Node v6.9.0, not Node latest.

File: Formula/kibana.rb (hunk 2 above)
Comment (ilovezfs) [inline Formula/kibana.rb]: We probably don't, but https://bot.brew.sh/job/Homebrew%20Core/12921/version=any_sierra/console and on 10.11 and 10.10 too. Most likely a one time fluke but I didn't want to end up with a CI dud again the second time through.

**Comment by jasontedor** [comment]
Thanks for the ping @ilovezfs, I left a review.

**Comment by ilovezfs** [comment]
@jasontedor thanks for your help!

**Comment by MikeMcQuaid** [comment]
We have `node@6` that we could/should probably use for this now.

**Comment by ilovezfs** [comment]
I think we should not depend on those.

**Comment by jasontedor** [comment]
I agree with @ilovezfs. The problem is if node@6 advances before Kibana has been tested against the advancement.

**Comment by ilovezfs** [comment]
:rocket:

**Comment by jasontedor** [comment]
Thanks @ilovezfs.

**Comment by ilovezfs** [comment]
@jasontedor you're welcome! Thanks again for your help and your review here.

**Pull request closed without merging**